promgen/templates/promgen/service_detail.html

```diff
@@ -8,15 +8,96 @@
 
 {% block content %}
 
-<div class="page-header" v-pre>
-  <h1>Service: {{ service.name }}</h1>
-  {% if service.owner %}
-  <p>{% trans 'Contact' %}: {{service.owner.username}}</p>
-  {% endif %}
+<div class="page-header promgen-flex-space-between-center">
+  <div>
+    <h1>Service: {{ service.name }}</h1>
+    {% if service.owner %}
+    <p>{% trans 'Contact' %}: {{service.owner.username}}</p>
```

Comment:
I'll review the rest in a bit, but in general we should probably write these out as `translate` now.

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#std:templatetag-translate

> The trans tag was renamed to translate. The trans tag is still supported as an alias for backwards compatibility.

```suggestion
    <p>{% translate 'Contact' %}: {{service.owner.username}}</p>
```

Reply:
Addressed this in [49aabd1](https://github.com/line/promgen/pull/520/commits/49aabd1298e8c90ad15205fdb0afadca98a0f27d)